packages/strapi-design-system/src/Tabs/TabGroup.tsx

```diff
@@ -1,14 +1,31 @@
 import * as React from 'react';
 
-import PropTypes from 'prop-types';
-
 import { TabsContext } from './TabsContext';
 import { useId } from '../hooks/useId';
 
+interface TabGroupProps {
+  children: React.ReactNode;
+  label: string;
+  id?: string;
+  initialSelectedTabIndex?: number;
+  onTabChange?: (index: number) => void;
+  variant?: 'simple';
+}
+
 export const TabGroup = React.forwardRef(
-  ({ id, initialSelectedTabIndex, label, onTabChange, variant, ...props }, ref) => {
+  ({ id, initialSelectedTabIndex, label, onTabChange = () => {}, variant, ...props }: TabGroupProps, ref) => {
     const tabsId = useId(id);
-    const Tabs = React.Children.toArray(props.children).find((node) => node.type.displayName === 'Tabs');
+
+    const Tabs = React.Children.toArray(props.children).find((child) => {
+      if (React.isValidElement(child)) {
+        const componentType = child.type as React.ComponentType<any>;
+
+        return componentType.displayName === 'Tabs';
+      }
+
+      return false;
+    }) as React.ReactElement | undefined;
```

Comment:
This is the only way I could find to make TypeScript happy happy...

But actually we never set a displayName on Tabs and doesn't seem to be inferred in anyway. So as far as I can tell this will always end up undefined and below on line 31 the condition will never pass. Maybe someone has some more context or understands better what's going on? Otherwise could we remove both?

Reply:
remove it. honestly it's awful and should never have been written in the first place.